### PR TITLE
Addition the menubar position in props

### PR DESCRIPTION
**Pull request by isv9**
Hi, I needed to set menubar in the bottom position and I did it by css order. Can you look it and if you think that is ok you will add it to your project. Thanks

### CODE REVIEW

File: src/styles/editor.scss [hunk 1]
```diff
@@ -25,6 +25,18 @@
...
+  .border-bottom {
+    border-bottom: 1px solid $lighter-border-color;
+  }
+
+  .order1 {
+    order: 1
+  }
+
+  .order2 {
+    order: 2
```
Comment (Leecason) [inline src/styles/editor.scss:37]: Codacy found an issue: [Expected a trailing semicolon (declaration-block-trailing-semicolon)](https://app.codacy.com/manual/Leecason/element-tiptap/pullRequest?prid=5779312)

File: src/styles/editor.scss [hunk 2]
```diff
@@ -25,6 +25,18 @@
     border-bottom-right-radius: 8px;
   }
 
+  .border-bottom {
+    border-bottom: 1px solid $lighter-border-color;
+  }
+
+  .order1 {
+    order: 1
```
Comment (Leecason) [inline src/styles/editor.scss:33]: Codacy found an issue: [Expected a trailing semicolon (declaration-block-trailing-semicolon)](https://app.codacy.com/manual/Leecason/element-tiptap/pullRequest?prid=5779312)

File: src/styles/editor.scss [hunk 3]
```diff
@@ -60,11 +72,17 @@
...
-    border-top: 0;
     color: $black-color;
     flex-grow: 1;
     padding: 30px 20px;
+
+    &_border-bottom_none{
+      border-bottom: 0;
+    }
+
+    &_border-top_none{
```
Comment (Leecason) [inline src/styles/editor.scss:83]: Codacy found an issue: [Expected single space before "{" (block-opening-brace-space-before)](https://app.codacy.com/manual/Leecason/element-tiptap/pullRequest?prid=5779312)

File: src/styles/editor.scss [hunk 4]
```diff
@@ -87,6 +105,12 @@
       right: 0;
       position: absolute;
     }
+
+    &_divider_none{
```
Comment (Leecason) [inline src/styles/editor.scss:109]: Codacy found an issue: [Expected single space before "{" (block-opening-brace-space-before)](https://app.codacy.com/manual/Leecason/element-tiptap/pullRequest?prid=5779312)

File: src/styles/editor.scss [hunk 5]
```diff
@@ -60,11 +72,17 @@
   > #{$root}__content {
     background-color: $white-color;
     border: 1px solid $lighter-border-color;
-    border-bottom: 0;
-    border-top: 0;
     color: $black-color;
     flex-grow: 1;
     padding: 30px 20px;
+
+    &_border-bottom_none{
```
Comment (Leecason) [inline src/styles/editor.scss:79]: Codacy found an issue: [Expected single space before "{" (block-opening-brace-space-before)](https://app.codacy.com/manual/Leecason/element-tiptap/pullRequest?prid=5779312)

File: src/styles/editor.scss (hunk 1 above)
Comment (isv9) [reply src/styles/editor.scss]: I've fixed it

File: src/styles/editor.scss (hunk 3 above)
Comment (isv9) [reply src/styles/editor.scss]: I've fixed it

File: src/styles/editor.scss (hunk 5 above)
Comment (isv9) [reply src/styles/editor.scss]: I've fixed it

**Comment by Leecason** [comment]
Thank you for your pr❤️.
I think `editorClass`, `editorContentClass`, `editorMenubarClass` props also can reposition the `menubar`, instead of `menubarPosition `. I don't want to add `menubarPosition` prop, it may be difficult to  add `left` and `right` menubar in the future.

**Comment by isv9** [comment]
You can add menubarPosition with ['top', 'bottom'] only and don't add 'left' and 'right'

**Comment by Leecason** [comment]
Does `editorClass`, `editorContentClass`, `editorMenubarClass` can implement `position menubar to bottom`? At the moment I don't want to add `menubarPosition` prop to element-tiptap.

**Comment by Leecason** [comment]
I add `editorClass`, `editorContentClass`, `editorMenubarClass`, `editorFooterClass` props, is these help you?

**Comment by isv9** [comment]
yes, thanks

**Pull request closed without merging**